src/components/VFilters/VSearchGridFilter.vue

```diff
@@ -107,8 +102,18 @@ export default {
       filterTypeTitle,
       clearFilters,
       onUpdateFilter,
-      isMinScreenMd,
     }
   },
 }
 </script>
+<style scoped>
+.filters {
+  border-inline-start: 1px solid transparent;
+}
+@media (min-width: 768px) {
```

Comment:
Tailwind's `@screen` might be better here.
```suggestion
@screen md {
```